ui/components/app/add-network/index.scss

```diff
@@ -51,6 +75,21 @@
       color: var(--color-text-alternative);
     }
   }
+
+  &__edge-case-box {
+    border: 1px solid var(--brand-colors-grey-grey100);
```

Comment:
@georgewrmarshall Can we use this color or should it have a different named color here?

Reply:
Should be using one of the border color css variables. I would suggest border-muted
```suggestion
    border: 1px solid var(--color-border-muted);
```

<img width="1436" alt="Screen Shot 2022-05-14 at 11 32 10 AM" src="https://user-images.githubusercontent.com/8112138/168444202-373b1a2f-0bef-4158-91dc-9b72373bb789.png">

Reply:
Fixed in: dc67e851ff3027e78e329448e64b8f8310629bf8